File: 1960-check-if-the-sentence-is-pangram/1960-check-if-the-sentence-is-pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        if (sentence.length() < 26) return false;

        boolean[] isPangram = new boolean[26];
        int cnt = 0;

        for (char c : sentence.toCharArray()) {
            int idx = c - 'a';
            if (!isPangram[idx]) {
                isPangram[idx] = true;
                cnt++;
                if (cnt == 26) return true;
            }
        }

        return false;
    }
}

File: 1960-check-if-the-sentence-is-pangram/1960-check-if-the-sentence-is-pangram.kt
class Solution {
    fun checkIfPangram(sentence: String): Boolean {
        if (sentence.length < 26) return false

        val isPangram = BooleanArray(26)
        var cnt = 0

        sentence.forEach { c ->
            val idx = c - 'a'
            if (!isPangram[idx]) {
                isPangram[idx] = true
                if (++cnt == 26) return true
            }
        }

        return false
    }
}